src/processes.c

```diff
@@ -1424,8 +1600,8 @@ static int ps_read_process(long pid, process_entry_t *ps, char *state) {
     return 0;
   }
 
-  cpu_user_counter = atoll(fields[11]);
-  cpu_system_counter = atoll(fields[12]);
+  long long cpu_user_counter = atoll(fields[11]);
+  long long cpu_system_counter = atoll(fields[12]);
```

Comment:
line 1576 could have comment like:
`// field numbers are ones listed for 'stat' file in 'man proc' - 3`